views/ideas.html

```diff
@@ -17,7 +17,18 @@ <h2>{{ .Name }}</h2>
 {{ end }}
 {{ end }}
 
-    <h1 class=span>Ideas</h1>
+    <h1 class=span>Languages</h1>
+
+{{ range .Data.Languages }}
+    <a class="card color" href="//github.com/code-golf/code-golf/issues/{{ .ID }}" title="{{ .Title }}">
+        <h2 class=span>{{ .Title }}</h2>
+        <svg><use href="#thumbs-up"/></svg>
+        {{ .ThumbsUp }}
+        <svg><use href="#thumbs-down"/></svg>
+        {{ .ThumbsDown }}
+    </a>
+{{ end }}
+    <h1 class=span>Other ideas</h1>
```

Comment:
title case please, Other Ideas, also maybe it should go first? there's generally fewer other ideas

Reply:
Yeah, I thought about putting Other Ideas first, but it didn't really make sense to me to call it "Other" then.